docs(tproxy): improve tproxy instructions (#1527)
kumahq/kuma-website: app/_src/production/dp-config/transparent-proxying.md

{{site.mesh_product_name}} comes with [`kumactl` executable](/docs/{{ page.version }}/explore/cli) which can help us to prepare the host. Due to the wide variety of Linux setup options, these steps may vary and may need to be adjusted for the specifics of the particular deployment.
The host that will run the `kuma-dp` process in transparent proxying mode needs to be prepared with the following steps executed as `root`:

1. Use proper version of iptables
    

>     {{site.mesh_product_name}} [isn't yet compatible](https://github.com/kumahq/kuma/issues/8293) with `nf_tables`. You can check the version of iptables with the following command

Reviewer: Not valid suggestion

    ```sh

>     $ iptables --version
>     iptables v1.8.7 (nf_tables)

Reviewer: ```suggestion
    iptables --version
    # iptables v1.8.7 (nf_tables)
```
It allows to copy and paste the code block without need to remove anything before running it in shell

    ```
    
    On the recent versions of Ubuntu, you need to change default `iptables`.
    

>     ```sh
>     $ update-alternatives --set iptables /usr/sbin/iptables-legacy
>     $ update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
>     $ iptables --version
>     iptables v1.8.7 (legacy)
> ```

Reviewer: ```suggestion
    ```sh
    update-alternatives --set iptables /usr/sbin/iptables-legacy
    update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
    iptables --version
    # iptables v1.8.7 (legacy)
    ```
```
It allows to copy and paste the code block without need to remove anything before running it in shell

### Upgrades

Before upgrading to the next version of {{site.mesh_product_name}}, it's best to clean existing `iptables` rules and only then replace the `kumactl` binary.

You can clean the rules either by restarting the host or by running following commands

> ```sh
> iptables -t nat -F
> iptables -t nat -X
> iptables -t raw -F
> iptables -t raw -X
> ip6tables -t nat -F
> ip6tables -t nat -X
> ip6tables -t raw -F
> ip6tables -t raw -X
> ```
> Be aware that this command removes all iptables rules, not only created by {{site.mesh_product_name}}. 

Reviewer: ```suggestion

{% warning %}
Executing these commands will remove all iptables rules, including those created by {{site.mesh_product_name}} and any other applications or services.
{% endwarning %}

```sh
iptables --table nat --flush
iptables --table raw --flush
ip6tables --table nat --flush
ip6tables --table raw --flush
iptables --table nat --delete-chain
iptables --table raw --delete-chain
ip6tables --table nat --delete-chain
ip6tables --table raw --delete-chain
```

```
